templates/kubeflow/features.html

```diff
@@ -7,17 +7,19 @@
 
 {% block content %}
 
-<section class="p-strip is-bordered">
-  <div class="row u-equal-height">
-    <div class="col-7">
-      <h1>Canonical&rsquo;s AI and ML solutions feature&hellip;</h1>
-      <p class="p-heading--three">Architectural freedom. Fully automated operations. Accelerated Deep Learning</p>
-      <p>Canonical&rsquo;s AI solutions such as <a href="/kubeflow/install">Kubeflow on Ubuntu</a> use your existing on-premise clusters and GPGPUs efficiently, giving you architectural freedom with storage and networking while sharing operational code with a large community. From your developer workstation, to your racks, to the public cloud, AI on Ubuntu is accelerated with the latest tools, drivers and libraries.</p>
-      <p>The standard for enterprise machine learning, from Silicon Valley to Wall Street, for the Fortune 50 and for startups.</p>
-      <p><a class="js-invoke-modal" href="/kubeflow/contact-us?product=ai-features">Contact us for machine learning, deep learning and AI consulting&nbsp;&rsaquo;</a></p>
-    </div>
-    <div class="col-4 col-start-large-9 u-vertically-center u-align--center u-hide--small">
-      <img src="https://assets.ubuntu.com/v1/7d33e0a1-AI+illustration+hero.svg" alt="Kubeflow" width="323" />
+<section class="p-strip is-bordered u-no-padding--top u-no-padding--bottom">
```

Comment:
You can add the `p-strip--suru-topped` to line 10 and remove `p-strip u-no-padding--top u-no-padding--bottom`. And also you can remove the next level `div` (line 11)
```suggestion
<section class="p-strip--suru-topped is-bordered ">
```

Reply:
Done